scripts/integration-tests/e2e-babel.sh

```diff
@@ -33,6 +33,10 @@ startLocalRegistry "$PWD"/scripts/integration-tests/verdaccio-config.yml
 node "$PWD"/scripts/integration-tests/utils/bump-babel-dependencies.js
 
 # Build and test
-YARN_ENABLE_IMMUTABLE_INSTALLS=false make -j test-ci
+YARN_ENABLE_IMMUTABLE_INSTALLS=false make -j build-standalone-ci
+for i in {1..50}; do
+  echo "RUN NUMBER $i"
+  BABEL_ENV=test yarn jest --maxWorkers=4 --ci
+done
```

Comment:
```suggestion
YARN_ENABLE_IMMUTABLE_INSTALLS=false make -j test-ci
```

(I'll commit this when the current CI run finishes)